versions/strategyMatrix.py

```diff
@@ -84,6 +85,14 @@ def rpm_package_details(pkg):
 f"https://nr-downloads-main.s3.amazonaws.com/infrastructure_agent/linux/yum/{rpm_os_family_remap[pkg['osDistro']]}/{pkg['osVersion']}/{pkg['arch']}/fluent-bit-{pkg['fbVersion']}-1.{pkg['osDistro']}-{pkg['osVersion']}.{rpm_target_arch_remap[pkg['arch']]}.rpm"
     }
 
+def sles_package_details(pkg):
+    return {
+        # SLES packages are not officially available in Fluent Bit repos (we compile them ourselves), so no 'packageUrl' is available for them.
+        'targetPackageName': f"fluent-bit-{pkg['fbVersion']}-1.{pkg['osDistro']}{pkg['osVersion']}.{pkg['arch']}.rpm",
```

Comment:
An example name can be found [here](https://github.com/newrelic/fluent-bit-package/releases/tag/2.0.8): `fluent-bit-2.0.8-1.sles12.4.x86_64.rpm`. In this case, there is no `- ` between `osDistro` and `osVersion` 🤷